locksmith/migrations/20230405185518-change-integer-to-big-integer.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    return Promise.all([
+      queryInterface.changeColumn('Transactions', 'chain', {
```

Comment:
Are we doing this for tables which are not used anymore? If so, for consistency - there is also `Blocks`.

Reply:
Good catch! Let me add `Blocks`! (and yes for consistency!).

Reply:
I can't because it is a primary key... also not a huge deal since we don't use this anymore?